api/models/team.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const teamSchema = mongoose.Schema({
+	_id: mongoose.Schema.Types.ObjectId,
```

Comment:
Do we really need an id? If this were an RDBMS, I'd say we need a primary key, but in a non-sql system, I'm not sure it adds any value.

Reply:
required by Mongo. if we dont explicitly create it, Mongo will create under the hood anyway following this naming convention "_id"